docs/machine-learning/resources/transforms.md

```diff
@@ -1,168 +1,103 @@
 ---
-title: Machine learning data transforms - ML.NET
+title: Data transformations | ML.NET
 description: Explore the feature engineering components supported in ML.NET.
-author: JRAlexander
-ms.custom: seodec18
-ms.date: 01/14/2019
+author: nake
+ms.date: 04/02/2019
 ---
 
-# Machine learning data transforms - ML.NET
+# Data transformations
 
-The following tables contain information about all of the data transforms supported in ML.NET.
+Chain data transformations together to prepare data for model training. All of the transformations in this guide return classes that implement the `IEstimator` interface. The specific implementation determines the type and format of data that the transformer expects from the previous element in the chain, and also the type and format of data that is produced by the transformation.
```

Comment:
1. Although transforms can and most likely will be chained together, it is not always the case. Start off by introducing transformations and then mention that they can be chained together. 
2. Add xref to `IEstimator`.

Reply:
Maybe something like "Data transformations are used to prepare data for model training. The transformations in this guide return classes that implement the [IEstimator interface](Microsoft.ML.IEstimator%601). Data transformations may also be chained together. When chained, the specific data transformation class determines the data type and format the transformer expects from the previous element in the chain, and also the data type and format the transformation produces. "